src/Symfony/Component/HttpFoundation/Tests/BinaryFileResponseTest.php

```diff
@@ -206,6 +206,21 @@ public function provideFullFileRanges()
         );
     }
 
+    public function testUnpreparedResponseSendsFullFile()
+    {
+        $response = BinaryFileResponse::create(__DIR__.'/File/Fixtures/test.gif', 200);
+
+        $file = fopen(__DIR__.'/File/Fixtures/test.gif', 'r');
```

Comment:
I'd suggest to use file_get_contents here

Reply:
Thanks. Updated.